idaes_examples/notebooks/docs/unit_models/custom_unit_models/liquid_extraction/liq_liq_extractor_flowsheet.py

```diff
@@ -0,0 +1,93 @@
+#################################################################################
+# The Institute for the Design of Advanced Energy Systems Integrated Platform
+# Framework (IDAES IP) was produced under the DOE Institute for the
+# Design of Advanced Energy Systems (IDAES).
+#
+# Copyright (c) 2018-2023 by the software owners: The Regents of the
+# University of California, through Lawrence Berkeley National Laboratory,
+# National Technology & Engineering Solutions of Sandia, LLC, Carnegie Mellon
+# University, West Virginia University Research Corporation, et al.
+# All rights reserved.  Please see the files COPYRIGHT.md and LICENSE.md
+# for full copyright and license information.
+#################################################################################
+'''
+    The below is an implementation of a flowsheet for liquid liquid extractor. 
+    The unit model uses two property packages for two liquid fluids
+'''
+import pyomo.environ as pyo
+import idaes.core
+import idaes.models.unit_models
+from idaes.core.solvers import get_solver
+import idaes.logger as idaeslog
+from pyomo.network import Arc
+from idaes.core.util.model_statistics import degrees_of_freedom
+from idaes.core.initialization import InitializationStatus
+from idaes.core.initialization.block_triangularization import (
+    BlockTriangularizationInitializer,
+)
+from idaes.core import FlowsheetBlock
+from idaes_examples.notebooks.docs.unit_models.custom_unit_models.liquid_extraction.organic_property import OrgPhase
+from aqueous_property import AqPhase
+from liquid_liquid_extractor import LiqExtraction
+
+
+def build_model(m):
```

Comment:
Is this code necessary? This look more like it would be suited as a test for the LEX model.